app/containers/Applications/ApplicationWizardConfirmation.tsx

```diff
@@ -306,9 +306,9 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
   return (
     <>
       <h1>Summary of your application:</h1>
-      <h5>
+      <p style={{fontSize: '1.25rem', fontWeight: 500}}>
         Please review the information you have provided before continuing.
-      </h5>
+      </p>
```

Comment:
Rule: Semantic/hierarchical Headings.

Changed to a paragraph because the text content doesn't make semantic sense as a heading. 

(ie: you wouldn't use this text to denote a page section that can be navigated to directly in a screenreader. The heading "Summary of your application" just above it would be used instead).